padinfo/id_menu.py

```diff
@@ -117,7 +118,10 @@ async def respond_with_refresh(message: Optional[Message], ims, **data):
 
     @staticmethod
     async def respond_with_delete(message: Optional[Message], ims, **data):
-        if ims.get('is_child') == 'True':
+        if ims.get('is_child'):
```

Comment:
idt you should do this in IdMenu. This logic should be controlled in a component that takes in an IdMenu + Disambig menu